### PR TITLE
PyTorch documentation updates

### PR DESCRIPTION
**Pull request by zredeaux65**
Corrected typos and sentence structure.

## What does this PR do?

Fixes #\<issue_number>

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/starter/introduction_guide.rst [hunk 1]
```diff
@@ -99,7 +99,7 @@ Let's first start with the model. In this case, we'll design a 3-layer neural ne
             return x
 
 Notice this is a :doc:`lightning module <../common/lightning_module>` instead of a ``torch.nn.Module``. A LightningModule is
-equivalent to a pure PyTorch Module except it has added functionality. However, you can use it **EXACTLY** the same as you would a PyTorch Module.
+equivalent to a pure PyTorch module except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch module.
```
Comment (ethanwharris) [inline docs/source/starter/introduction_guide.rst:102]: Would maybe keep `Module` as it is a class. Perhaps it should be:
```suggestion
equivalent to a pure PyTorch ``nn.Module`` except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch ``nn.Module``.
```
Otherwise could just have:
```suggestion
equivalent to a pure PyTorch ``Module`` except it has added functionality. However, you can use it **exactly** the same as you would a PyTorch ``Module``.
```

File: docs/source/starter/lightning_lite.rst [hunk 2]
```diff
@@ -416,7 +416,7 @@ Configure the devices to run on. Can be of type:
     # equivalent
     lite = Lite(devices=0)
 
-    # int: run on 2 GPUs
+    # int: run on to GPUs
```
Comment (ethanwharris) [inline docs/source/starter/lightning_lite.rst:419]:
```suggestion
    # int: run on two GPUs
```

File: docs/source/starter/new-project.rst [hunk 3]
```diff
@@ -384,7 +381,7 @@ You can also add a forward method to do predictions however you want.
 .. code-block:: python
 
     # ----------------------------------
```
Comment (ethanwharris) [inline docs/source/starter/new-project.rst:383]:
```suggestion
    # -------------------------------
```

File: docs/source/starter/new-project.rst [hunk 4]
```diff
@@ -384,7 +381,7 @@ You can also add a forward method to do predictions however you want.
...
-    # or using the AE to generate images
+    # using the AE to generate images
     # ----------------------------------
```
Comment (ethanwharris) [inline docs/source/starter/new-project.rst:385]:
```suggestion
    # -------------------------------
```

**Review by ethanwharris — COMMENTED** [review]
This is awesome! Few comments :smiley:

**Review by awaelchli — APPROVED** [review]
love it

File: docs/source/starter/introduction_guide.rst [hunk 5]
```diff
@@ -1003,15 +1002,15 @@ Child Modules
...
+**********************
+Why PyTorch Lightning?
+**********************
 
 a. Less boilerplate
 ===================
 
 Research and production code starts with simple code, but quickly grows in complexity
-once you add GPU training, 16-bit, checkpointing, logging, etc...
+once you add GPU training, 16-bit, checkpointing, logging, and so on.
```
[the remark this answers is not in the document]
Comment (tchaton) [reply docs/source/starter/introduction_guide.rst:1013]: It is more common I guess :)

File: docs/source/starter/lightning_lite.rst [hunk 6]
```diff
@@ -221,28 +221,28 @@ but there are several major challenges ahead of you now:
    :header-rows: 0
 
    * - Processes divergence
-     - This happens when processes execute a different section of the code due to different if/else conditions, race condition on existing files, etc., resulting in hanging.
+     - This happens when processes execute a different section of the code due to different if/else conditions, race conditions on existing files, etc., resulting in hanging.
    * - Cross processes reduction
```
Comment (tchaton) [inline docs/source/starter/lightning_lite.rst:225]:
```suggestion
     - This happens when processes execute a different section of the code due to different if/else conditions, race conditions on existing files and so on, resulting in hanging.
   * - Cross processes reduction
```

**Review by tchaton — APPROVED** [review]
Wow ! Really impressive work !

**Review by s-rog — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

File: docs/source/starter/lightning_lite.rst [hunk 7]
```diff
@@ -694,8 +694,8 @@ handling the saving part correctly, no matter if you are running single device,
 load
 ====
 
-Load checkpoint contents from a file. Replaces all occurences of ``torch.load(...)`` in your code. Lite will take care of
-handling the loading part correctly, no matter if you are running single device, multi-device or multi-node.
+Load checkpoint contents from a file. Replaces all occurrences of ``torch.load(...)`` in your code. Lite will take care of
+handling the loading part correctly, no matter if you are running a single device, multi-devices or multi-nodes.
```
Comment (rohitgr7) [inline docs/source/starter/lightning_lite.rst:698]:
```suggestion
handling the loading part correctly, no matter if you are running a single device, multi-device, or multi-node.
```
I think it's just multi-node technically.

**Review by rohitgr7 — APPROVED** [review]
really amazing improvements 😃 
thank you

**Review by SkafteNicki — APPROVED** [review]

**Review by Borda — APPROVED** [review]
